types/accessRequestSearchResponse.js

```diff
@@ -0,0 +1,60 @@
+const AccessRequest = require('./accessRequest')
+const Serializable = require('./serializable')
+
+class AccessRequestSearchResponse extends Serializable {
+  constructor(accessRequests, nextToken) {
+    super()
+    this.accessRequests = accessRequests
+    this.nextToken = nextToken
+  }
+
+  serializable() {
+    let serializedAccessRequests = []
+    for (const accessRequest of this.accessRequests) {
+      serializedAccessRequests.push(accessRequest.serializable())
+    }
+    let toSerialize = {
+      access_requests: serializedAccessRequests,
+      next_token: this.nextToken,
+    }
+    const serializedKeys = Object.keys(toSerialize)
+    for (const key of serializedKeys) {
+      if (toSerialize[key] === null) {
+        delete toSerialize[key]
+      }
+    }
+    return toSerialize
+  }
+  static async decode(json) {
```

Comment:
this doesn't need to be `async`

Reply:
it does now (I had to do some async await things to get the using of the decode on AccessRequest to work)